src/components/Battles.test.ts

```diff
@@ -0,0 +1,115 @@
+import { expect } from "chai";
+import { Clock } from "lolex";
+import { MenuGraphr } from "menugraphr";
+import { FullScreenPokemon } from "..";
+import { createBattleMover } from "../creators/createBattleMover";
+import { stubBlankGame } from "../fakes.test";
+import { IBattleTeam, IPartialBattleOptions, IPokemon } from "./Battles";
+import { IPlayer } from "./Things";
+
+const createGame = () => {
+    const { clock, fsp, player } = stubBlankGame(
+        {height: 1000,
```

Comment:
Interesting `{` style; I haven't seen this one before. Here and elsewhere, keep it to the existing style

```typescript
methodCall({
    foo: "bar",
});
```

_(next semester I think I'll have Prettier set up, but until then...)_